src/Components/SceneList/SceneList.tsx

```diff
@@ -207,6 +208,9 @@ const SceneList: React.FC<SceneListProps> = ({
                                 event.stopPropagation();
                                 setSelectedScene(item);
                                 setIsSceneDialogOpen(true);
+                                useTelemetry().sendEventTelemetry({
```

Comment:
instead of calling useTelemetry().sendEventTelemetry multiple times as below can we do as we do for useTranslation?:
`const { sendEventTelemetry } = useTelemetry();` and then calling just sendEventTelemetry?

Reply:
Sure. Done.